controller/src/main/java/io/pravega/controller/store/client/StoreClientFactory.java

```diff
@@ -62,6 +70,20 @@ private static CuratorFramework createZKClient(ZKClientConfig zkClientConfig) {
         return zkClient;
     }
 
+    private static String loadPasswdFromFile(String trustStorePasswordPath) {
+        byte[] pwd;
+        File passwdFile = new File(trustStorePasswordPath);
+        if (passwdFile.length() == 0) {
+            return "";
+        }
+        try {
+            pwd = FileUtils.readFileToByteArray(passwdFile);
```

Comment:
I made this comment in a different PR. Please add a maximum file length check here. I don't want to crash the process if someone passes in a 2GB file.